src/aws/dynamodb-utils.ts

```diff
@@ -15,14 +15,16 @@ import {
   TransactWriteItemsInput,
 } from 'aws-sdk/clients/dynamodb';
 import AttributeValue = DocumentClient.AttributeValue;
-const dynamodb: DynamoDbClient = require('serverless-dynamodb-client');
+// const dynamodb: DynamoDbClient = require('serverless-dynamodb-client');
```

Comment:
nit: Can delete?

Reply:
nah, need to revert this. shouldn't have added it